shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1320,7 +1320,7 @@ object GlutenConfig {
         "Experimental: The class name of user-defined cost model that will be used by RAS. " +
           "If not specified, a rough built-in cost model will be used.")
```

Comment:
nit: "If not specified, a legacy built-in cost model that exhaustively offloads computations will be used"